test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -652,9 +652,9 @@ func runWorkload(b *testing.B, tc *testCase, w *workload) []DataItem {
 					b.Fatalf("failed to create namespace for Pod: %v", namespace)
 				}
 				b.Cleanup(func() {
-					err := client.CoreV1().Namespaces().Delete(ctx, namespace, metav1.DeleteOptions{})
+					err := client.CoreV1().Namespaces().Delete(context.TODO(), namespace, metav1.DeleteOptions{})
```

Comment:
The creation code tolerates already existing namespaces. Can that really happen?

If the code is called twice for the same namespace name, then it will only be created once, but two cleanup functions will be registered. Only the first one will succeed, the second or later ones will fail with a NotFound. That error needs to be ignored.

I don't know how many namespaces need to be created. If it's just a few, then the current approach with trying the operation and ignoring acceptable errors (in both places!) is okay. If it's many, then a `var namespaces map[string]bool` outside of the for loop could track which namespaces were already created. Only one cleanup function is needed then, registered outside of the loop, which then iterates over the created namespaces.

Reply:
> The creation code tolerates already existing namespaces. Can that really happen?

Users can create namespace with `createNamespace` operation. So, it may happen.

> If the code is called twice for the same namespace name, then it will only be created once, but two cleanup functions will be registered. Only the first one will succeed, the second or later ones will fail with a NotFound. That error needs to be ignored.

It's a fair point. I think it's enough to change it to register only when no error from the creation of namespace.

Reply:
Right, that's simpler.